tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -4099,58 +4099,37 @@ def get_mod(a):
         b = mod(a, a)
         return b
 
-    integer8 = randint(min_int8, max_int8, dtype=np.int8)
-    integer16 = randint(min_int16, max_int16, dtype=np.int16)
-    integer = randint(min_int, max_int, dtype=int)
-    integer32 = randint(min_int32, max_int32, dtype=np.int32)
-    integer64 = randint(min_int64, max_int64, dtype=np.int64)
-
-    fl = uniform(min_float / 2, max_float / 2)
-    fl32 = uniform(min_float32 / 2, max_float32 / 2)
-    fl32 = np.float32(fl32)
-    fl64 = uniform(min_float64 / 2, max_float64 / 2)
-
-
     epyccel_func = epyccel(get_mod, language=language)
 
     f_bl_true_output = epyccel_func(True)
     test_bool_true_output = get_mod(True)
 
     assert f_bl_true_output == test_bool_true_output
-
     assert matching_types(f_bl_true_output, test_bool_true_output)
 
-    f_integer_output = epyccel_func(integer)
-    test_int_output  = get_mod(integer)
+    def test_int(min_int, max_int, dtype):
+        integer = randint(min_int, max_int, dtype=dtype)
+        if 0 == integer:
+            integer += 1
```

Comment:
Can't you just write this?
```python
integer = randint(min_int, max_int, dtype=dtype) or 1
```

Reply:
You are right, I am not used to this syntax. Fixed in 5bd7230